core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReader.java

```diff
@@ -71,9 +72,8 @@ protected TabletServerBatchReader(ClientContext context, Class<?> scopeClass, Ta
     this.tableName = tableName;
     this.numThreads = numQueryThreads;
 
-    queryThreadPool =
-        context.threadPools().getPoolBuilder("batch scanner " + batchReaderInstance + "-")
-            .numCoreThreads(numQueryThreads).build();
+    queryThreadPool = context.threadPools().getPoolBuilder(BATCH_SCANNER_POOL_NAME)
```

Comment:
are we losing the batch reader instance in the thread pool name?

Reply:
Yes, the batch Reader instance is coming from the atomic counter. I'll modify this to append the scanner instance number.

Reply:
fixed in 3bcc3f8854